Mathlib/Order/Max.lean

```diff
@@ -45,6 +45,7 @@ class NoBotOrder (α : Type*) [LE α] : Prop where
   exists_not_ge (a : α) : ∃ b, ¬a ≤ b
 
 /-- Order without top elements. -/
+@[to_dual]
```

Comment:
This is a general question: I seem to remember that `existing` was required when the target decl already exists, but clearly not. Did something change here or am I misremembering?